WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsModel.cs

```diff
@@ -269,7 +249,7 @@ private async Task<List<ChangeAvoidanceSuggestion>> CreateChangeAvoidanceSuggest
 		// If the original transaction couldn't avoid the CJing coins, BnB can use them too. Otherwise exclude them.
 		coinsToUse = spentCoins.Any(coinsToExclude.Contains) ? coinsToUse : coinsToUse.Except(coinsToExclude).ToImmutableArray();
 
-		var suggestions = CreateChangeAvoidanceSuggestionsAsync(info, coinsToUse, maxInputCount, usdExchangeRate, linkedCts.Token);
+		var suggestions = CreateChangeAvoidanceSuggestionsAsync(info, coinsToUse, maxInputCount, usdExchangeRate, linkedCts.Token).ConfigureAwait(false);
```

Comment:
@soosr I got a new clue:

![image](https://github.com/zkSNACKs/WalletWasabi/assets/98904713/8475d9d4-d5e2-4a9b-8aa2-0e600288e8f6)

I need to figure out why it's getting a cancelled CTS instead of a fresh one